Mobile/ShopLens/Droid/Activities/CameraActivity.cs

```diff
@@ -77,11 +75,8 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             if (IsThereAnAppToTakePictures())
             {
-                // Set up a custom speech recognizer in this activity.
-                commandRecognizer = SpeechRecognizer.CreateSpeechRecognizer(this);
```

Comment:
I didn't really do anything with Camera activity, because it's gonna be merged with Main activity by Tomas. I just removed the voice recognizer methods.